CHANGELOG.md

```diff
@@ -339,6 +339,10 @@ This release brings further improvements to Calendar types. It adds arithmetic a
 
   * [EEx] Deprecate `<%= ` in "middle" and "end" expressions, e.g.: `<%= else %>` and `<%= end %>`
 
+#### ExUnit
```

Comment:
Happy to change it. It didn't really like a bug for me, that's why I used deprecation. As in, it wasn't entirely clear to me that it was the behaviour people expected (the impossibility of having multiple `describe` blocks with the same name).